### PR TITLE
Allow '0' in `decimals` option for `NumberField` and add test for negative decimals

### PR DESCRIPTION
**Pull request by michael-toporkov**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+3.21.3
+======
+
+* (bug) Allow '0' in `decimals` option for `NumberField`
```
Comment (apfelbox) [inline CHANGELOG.md]:
```suggestion
* (bug) Properly support '0' in `decimals` and `minValue` option for `NumberField`
```
